src/components/pages/login/LoginForm.js

```diff
@@ -1,18 +1,21 @@
 import React, { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+
 
 export default function LoginForm() {
     // State
-    const [name, setName] = useState("");
+    const [name, setName] = useState("")
+    const navigate = useNavigate()
+
 
     // Comportements
     const handleChange = (e) => {
         setName(e.target.value)
     }
 
     const handleSumbit = (e) => {
+        navigate("order/" + name)
         e.preventDefault();
```

Comment:
```suggestion
        e.preventDefault();
        navigate("order/" + name)
```

On veut **d'abord** empêcher le rechargement de la page **et ensuite** exécuter les instructions du comportement handleSubmit, dont celui de rediriger vers la page OrderPage